lib/haskell/natural4/natural4.cabal

```diff
@@ -159,7 +159,7 @@ library
     , mtl
     , nonempty-containers
     , optics
-    , optparse-generic
+    , optparse-applicative
```

Comment:
Incidentally, it may be worth adding a quick comment somewhere that explains why we should just stick to `optparse-applicative` instead of using `optparse-generic`, if this hasn't been documented already.